src/BinSkim.Rules/PERules/BA2025.EnableShadowStack.cs

```diff
@@ -60,6 +60,12 @@ public override AnalysisApplicability CanAnalyzePE(PEBinary target, Sarif.Proper
                 return notApplicable;
             }
 
+            if (portableExecutable.IsARM64Binary)
```

Comment:
>portableExecutable.IsARM64Binary [](http://example.com/codeflow?start=16&length=32)

So, you'll change this to portableExecutable.Machine == Machine.Arm64...

Reply:
Nice! btw - if you feel like these issues are resolved, please feel free to mark them that way.